routes/service.php

```diff
@@ -102,6 +102,14 @@
         'as' => 'admin.centres.store',
         'uses' => 'Admin\CentresController@store',
     ]);
+    Route::post('centres/update', [
```

Comment:
Semantically, a PUT or PATCH to /centres with a single item is an update of the centres collection
OR
a PUT to  `centres/{id}/update`, would be more explicit, so we know which ID we're updating without having to grab it from the input (See above).

Reply:
Updated as requested.